pkg/nsx/services/common/types.go

```diff
@@ -60,6 +60,8 @@ const (
 	TagScopeVPCManagedBy               string = "nsx/managed-by"
 	AutoCreatedVPCTagValue             string = "nsx-op"
 	LabelDefaultSubnetSet              string = "nsxoperator.vmware.com/default-subnetset-for"
+	LabelImageFetcher                  string = "wcp.vmware.com/image-fetcher"
```

Comment:
now  changed to iaas.vmware.com/image-fetcher

Reply:
ok